server/tests/__mock__/sources.ts

```diff
@@ -6,8 +6,8 @@ import { getSources, Sources } from '../../src/sources'
 export function getMockSources(): Sources {
```

Comment:
Preferably, we'd have e2e tests - request Prisma/Algolia and check whether an instance was actually created or not.